spec/lib/vanagon/component/source/http_spec.rb

```diff
@@ -30,7 +30,7 @@
 
   describe "#get_extension" do
     it "returns the extension for valid extensions" do
-      (Vanagon::Component::Source::Http::ARCHIVE_EXTENSIONS + Vanagon::Component::Source::Http::NON_ARCHIVE_EXTENSIONS).each do |ext|
+      Vanagon::Component::Source::Http::ARCHIVE_EXTENSIONS.each do |ext|
```

Comment:
In light of the above comment about `txt` vs `.txt` it would be worth adding a test for non-archive extensions.